Sources/SwiftProtobuf/Google_Protobuf_Timestamp+Extensions.swift

```diff
@@ -217,6 +218,44 @@ extension Google_Protobuf_Timestamp: _CustomJSONCodable {
     }
 }
 
+public extension Google_Protobuf_Timestamp {
+    public init(timeIntervalSince1970: TimeInterval) {
```

Comment:
Do you want to go ahead and throw doc strings on these now so they are documented?